gelfling.js

```diff
@@ -49,7 +106,15 @@ Gelfling.prototype.send = function(data, callback) {
     udpClient.send(data[i], 0, data[i].length, this.port, this.host, checkDone)
 }
 
+Gelfling.prototype.send = function(data, callback) {
+  if (callback == null) callback = function() {}
+
+  var send = this.tcp ? this._sendTcp : this._sendUdp
+  send.call(this, this.convert(data), callback)
```

Comment:
Hmmm – I'm just looking at this and it looks like you're changing the signature for this?

`send` used to be able to accept a Buffer and an array of Buffers (as well as a message object which would be converted) – see https://github.com/mhart/gelfling/blob/master/gelfling.js#L25

Now you're calling `this.convert(data)` which will break in those cases.